ChangeLog.md

```diff
@@ -24,6 +24,8 @@ Current Trunk
   version of clang rather than replacing it.
 - ASAN_SHADOW_SIZE is deprecated. When using AddressSanitizer, the correct
   amount of shadow memory will now be calculated automatically.
+- IDBFS now persists files whenever their timestamp changes; previously it acted
+  on sync only if the timestamp increased and ignored the file changes otherwise.
 
```

Comment:
we tagged 2.0.6 meanwhile, so this should go in `Current Trunk`.

Reply:
(side note, adding to the top of the list avoids this being a silent error, but then you'd get a merge conflict - so you still need to do a manual operation here, no way around that sadly...)

Reply:
Done.